site/content/en/docs/Reference/gameserver.md

```diff
@@ -123,6 +123,10 @@ The `spec` field is the actual GameServer specification and it is composed as fo
 - `players` (Alpha, behind "PlayerTracking" feature gate), sets this GameServer's initial player capacity
 - `template` the [pod spec template]({{% k8s-api href="#podtemplatespec-v1-core" %}}) to run your GameServer containers, [see](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#pod-templates) for more information.
 
+{{< alert title="Note" color="info">}}
+Please note that the GameServer resource does not support updates. If you need to make regular updates to the GameServer spec, consider using a [Fleet]({{< ref "/docs/Reference/fleet.md" >}}).
```

Comment:
This is a mega tiny nit!

```suggestion
The GameServer resource does not support updates. If you need to make regular updates to the GameServer spec, consider using a [Fleet]({{< ref "/docs/Reference/fleet.md" >}}).
```

Because the title says "note" in it already, it's just a little redundant to write it twice.

Was just looking at the preview here:
https://47c8938-dot-preview-dot-agones-images.appspot.com/site/docs/reference/gameserver/

Other than that, this is good to go!

Reply:
Commited!